### DISCUSSION TITLE
Vercel Deployment

### DISCUSSION THREAD

**Discussion by MeldinRadoncic** [vercel/next.js#55145] [Help]
### Summary

Does anyone know why, when I push the changes to the main repository, Vercel does not receive that commit and does not redeploy? I had a project in the hobby plan, and I transferred the project to a Pro account. Now, when I commit the changes, nothing happens. Thank you.

**Comment by icyJoseph** [comment]
Could you have accidentally set this? https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel

**Comment by duoluodexiaoxiaoyuan** [comment]
> Could you have accidentally set this? https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel
excuse me , I meet a questions. I don't how to slove it. When I refresh the page, my interface is called 4 times and I don't know what the reason is.

app directionary -> layout.js
```javascript
import './globals.css'
import { Inter } from 'next/font/google'
import { Suspense } from 'react'
import Loading from '@/app/loading.js'

const inter = Inter({ subsets: ['latin'] })

export const metadata = {
  title: 'Create Next App',
  description: 'Generated by create next app',
}
// 默认写的是{children}，但是我们写成props的时候要把{}去掉
export default function RootLayout(props) {
  return (
    <html lang="en">
      <body className={inter.className}>
        {props.children}
      </body>
    </html>
  )
}

```

_lib -> EveryWord.js
```javascript
import React from 'react'

// const getEveryWord = () => {
//   return fetch(`https://v.api.aa1.cn/api/api-wenan-anwei/index.php?type=json`)	
//   .then(response => response.json())
// }
const getEveryWord = () => {
  return fetch(`http://localhost:12600/ziji`)	
  .then(response => response.json())
}

async function EveryWord() {
  
  const data =await getEveryWord()
  console.log(data)
  return (
    <div>EveryWords{data.name} </div>
  )
}

export default EveryWord
```

this is my github repo: https://github.com/duoluodexiaoxiaoyuan/mynextproject/tree/useRequestWithNext

if u look, thank u explain
